pkg/model/stack.go

```diff
@@ -68,8 +69,9 @@ type Service struct {
 }
 
 type StackVolume struct {
-	LocalPath  string
-	RemotePath string
+	LocalPath          string
+	RemotePath         string
+	isPersistentVolume bool
```

Comment:
what is `isPersistentVolume` for? they are always persistent, no?

Reply:
I need this to know if the volume is defined on the volume top level section or directly on the `service.volumes` section.

Reply:
is it possible to check if `volumeName` in `s.Volumes` instead when you need that check?